platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -330,18 +331,22 @@ private void wire() {
                         .getStreamSequenceNumber());
         keystoneEventSequenceNumberTransformer.getOutputWire().solderTo(pcesWriterWiring.flushRequestInputWire());
 
+        final OutputWire<ConsensusRound> consensusRoundOutputWire = consensusEngineWiring.getSplitOutput();
+
         // The request to flush the keystone event for a round must be sent to the PCES writer before the consensus
         // round is passed to the round handler. This prevents a deadlock scenario where the consensus round
         // handler has a full queue and won't accept additional rounds, and is waiting on a keystone event to be
         // durably flushed to disk. Meanwhile, the PCES writer hasn't even received the flush request yet, so the
         // necessary keystone event is *never* flushed.
-        consensusEngineWiring
-                .consensusRoundOutput()
-                .orderedSolderTo(List.of(
-                        keystoneEventSequenceNumberTransformer.getInputWire(),
-                        consensusRoundHandlerWiring.roundInput()));
-        consensusEngineWiring.consensusRoundOutput().solderTo(eventWindowManagerWiring.consensusRoundInput());
-        consensusEngineWiring.consensusEventsOutput().solderTo(eventStreamManagerWiring.eventsInput());
+        consensusRoundOutputWire.orderedSolderTo(List.of(
+                keystoneEventSequenceNumberTransformer.getInputWire(), consensusRoundHandlerWiring.roundInput()));
+        consensusRoundOutputWire.solderTo(eventWindowManagerWiring.consensusRoundInput());
+
+        // Future work: the framework will eventually support transforming split output.
```

Comment:
the PR is open for this, so it might even merge before this

Reply:
It's merged - I will update this PR to use the new feature and remove the future work comment

Reply:
Resolving this comment, since the update has been made and reviewed by Cody, and Ed, and @cody-littley has changes that are waiting for this PR to merge.